### ISSUE TITLE
None of my cameras work with howdy

### ISSUE THREAD

**Issue by mittsquared** [boltgolt/howdy#91]
First of all, thank you for your work on this project.

I went through the setup process and everything seemed fine. But when I go to register a face, none of my cameras work. I tested with all four of my `/dev/video*` devices (two of them turn on the IR emitters).

When running `sudo howdy test`, the RGB camera doesn't load at all (it works in Cheese, though). Here is the error:

```
X Error: BadDrawable (invalid Pixmap or Window parameter) 9
  Major opcode: 62 (X_CopyArea)
  Resource id:  0x3200010
```

The IR camera doesn't work in Cheese or Howdy. In Cheese, it freezes the viewfinder until I switch back to the RGB camera.

My laptop is a Samsung NP940X5M. All of the devices work fine in Windows. I believe the drivers for the IR camera are from Realtek.

----

Linux distribution (if applicable): Ubuntu 18.10

Howdy version: 2.3.1

**Comment by mittsquared** [comment]
Thank you for the fast response. Modifying `/etc/environment` fixed the QT issue; the RGB camera works with `sudo howdy test` now.

However, `dmesg | grep "Unknown video format"` outputs nothing. I scanned through `dmesg` manually and found this:

```
[    3.832298] input: 720p HD Camera: Samsung IR Came as /devices/pci0000:00/0000:00:14.0/usb1/1-6/1-6:1.2/input/input26
```

If I'm understanding this correctly, it looks like the IR camera is initialized but doesn't work for some reason.

**Comment by mittsquared** [comment]
After some more digging, I found an old post that suggested installing `guvcview` and testing the camera there. This worked! The camera reports the RGB resolution as the IR resolution, which is incorrect. The IR camera outputs at 340x340 (in my case). `howdy` was defaulting to the higher resolution, causing issues. Manually setting the res using `sudo howdy config` fixed the problem.

Thanks again for the project and your hard work. Closing this issue.
